reference/src/layout/pointers.md

```diff
@@ -31,6 +31,10 @@ and are at least one word.
 
 The layouts of `&T`, `&mut T` and `*T` are the same.
 
+If `T` is sized, references and pointers to `T` have a size and alignment of one
+word and have therefore the same layout as C pointers - they can be used in C FFI
+where pointers are expected.
```

Comment:
It might be worth mentioning that references come with a validity invariant that makes them *not* usable in FFI contexts whenever there might be NULL, unaligned or dangling pointers.

Reply:
I think i'll just leave the last part about usable in C FFI out. Stating that they have the same layout as C pointers is enough.

Reply:
Still feels like a warning would be prudent. IIRC we plan to eventually merge the validity and layout sections for each of these types, then such a warning would be even more natural.

Reply:
Done.